src/main/java/com/SAS/League/GamesList.java

```diff
@@ -0,0 +1,38 @@
+package com.SAS.League;
+
+import java.util.ArrayList;
+
+public class GamesList {
```

Comment:
I think this class is not necessary because there should be a trinary connection between - Game, Season and League.

Reply:
I also did not quite understand why this is necessary